<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txt1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>757</width>
      <height>515</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>3</horstretch>
      <verstretch>3</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>2</width>
      <height>2</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>700</width>
      <height>500</height>
     </size>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="tabStopDistance">
     <double>79.000000000000000</double>
    </property>
    <property name="placeholderText">
     <string>Scrivi qui</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSalva"/>
    <addaction name="actionApri"/>
   </widget>
   <widget class="QMenu" name="menuModifica">
    <property name="title">
     <string>Modifica</string>
    </property>
    <widget class="QMenu" name="menuFormattazione">
     <property name="title">
      <string>Formattazione</string>
     </property>
     <addaction name="actionGrassetto"/>
     <addaction name="actionCorsivo"/>
    </widget>
    <widget class="QMenu" name="menuColore">
     <property name="title">
      <string>Colore</string>
     </property>
     <addaction name="actionRosso"/>
     <addaction name="actionBlu"/>
     <addaction name="actionVerde"/>
     <addaction name="actionGiallo"/>
     <addaction name="actionnero"/>
     <addaction name="actionBianco"/>
     <addaction name="actionPersonalizzato"/>
    </widget>
    <addaction name="menuFormattazione"/>
    <addaction name="menuColore"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actioninfo"/>
    <addaction name="separator"/>
    <addaction name="actionEsci"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuModifica"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actioninfo">
   <property name="text">
    <string>info</string>
   </property>
  </action>
  <action name="actionEsci">
   <property name="text">
    <string>Esci</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSalva">
   <property name="text">
    <string>Salva</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionApri">
   <property name="text">
    <string>Apri</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGrassetto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grassetto</string>
   </property>
  </action>
  <action name="actionCorsivo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Corsivo</string>
   </property>
  </action>
  <action name="actionRosso">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rosso</string>
   </property>
  </action>
  <action name="actionBlu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blu</string>
   </property>
  </action>
  <action name="actionVerde">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verde</string>
   </property>
  </action>
  <action name="actionGiallo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Giallo</string>
   </property>
  </action>
  <action name="actionnero">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nero</string>
   </property>
  </action>
  <action name="actionBianco">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bianco</string>
   </property>
  </action>
  <action name="actionPersonalizzato">
   <property name="text">
    <string>Personalizzato</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
